src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -73,4 +73,10 @@ protected void initBinder(WebDataBinder binder) {
         List<SectionResponse> sections = lineService.getSections(lineId);
         return ResponseEntity.ok(sections);
     }
+
+    @DeleteMapping("/{id}/sections")
+    public ResponseEntity<?> deleteStation(@PathVariable("id") Long lineId, @RequestParam("stationId") Long stationId) {
```

Comment:
와일드 카드 대신에 Void를 사용할 수도 있을 것 같아요 :)
```suggestion
    public ResponseEntity<Void> deleteStation(@PathVariable("id") Long lineId, @RequestParam("stationId") Long stationId) {
```

Reply:
Void 타입으로 반환하면 되었군요!
꼼꼼히 봐주시는 덕분에 많이 배우고 있습니다 감사합니다!